src/content/_contributors.yaml

```diff
@@ -1065,3 +1065,21 @@ captainpangyo:
   email: jangkeehyo@google.com
   description:
     en: "Josh is a Singing Computer Scientist"
+
+
+moinshaikh:
+  name:
+    given: Moin
+    family: Shaikh
+  org:
+    name: N/A
+    unit: Developer / l10n contributor
+  country: IN
+  role:
+    - translator
+  homepage: https://moingshaikh.wordpress.com/
+  google: +MoinShaikhMS
+  twitter:moingshaikh
```

Comment:
Our YAML parser is failing because it needs a space between the attribute and the value, can you update and resubmit please?